### PR TITLE
Little documentation

### LINKED ISSUE #1356: Write a little documentation for these new MapKnitter Lite features
Something to think on - we should write up a very brief description of how this works -- linking to some examples and maybe an example .json file like https://archive.org/download/empire-st-chinatown-providence/mapknitter-1676395083251.json

Not urgent but to consider!

### PR DESCRIPTION
**Pull request by segun-codes**
Fixes #1356 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [X] PR is descriptively titled 📑 and links the original issue above 🔗
* [X] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt test`
* [X] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updates
* [X] @mention the original creator of the issue in a comment below for help or for a review

### CODE REVIEW

**Comment by segun-codes** [comment]
Hi @jywarren, @7malikk, here's the documentation. you may kindly review.

1. https://github.com/segun-codes/Leaflet.DistortableImage-arc/blob/prelim-documentation/examples/README.md
2. You will find this in the "Files Changed" tab.

**Comment by segun-codes** [comment]
@jywarren @7malikk, your review are still being awaited. Many thanks!

**Review by jywarren — APPROVED** [review]
This is really nice documentation! I'd like to make some text tweaks to it and add a commit here, but great job in documenting the various main features.

File: README.md [hunk 1]
```diff
@@ -726,6 +726,30 @@ A collection instance made up of a group of images. Images can be "collected" in
...
 
+Retrieve image from Json file containing image property set. The property set can be used to instantiate new imageOverlays.  
+<details><summary><code><b>recreateImagesFromJsonUrl(string)</b>: {avg_cm_per_pixel <string>, imgCollectionProps<{}>}</code></summary>
+<ul><li>Returns imageCollectionIbject if successful or empty object if unsuccessful</li></ul>
+
+Example<br>
+// 1. Instantiate an empty `distortableCollection`<br>
+<code>imgGroup = L.distortableCollection().addTo(map);</code>
+
+// 2. Get property set for each of the images<br>
```
Comment (jywarren) [inline README.md:737]: Nice separation of core library features from our application!

**Comment by segun-codes** [comment]
> This is really nice documentation! I'd like to make some text tweaks to it and add a commit here, but great job in documenting the various main features.

Yes, please. You can go ahead...

**Comment by jywarren** [comment]
OK, sorry that took me a while! I also better understand the question about pointing at github pages vs. the local copy. I opted for GH Pages, so that it's more immediately usable!

**Comment by jywarren** [comment]
OK, I think we can merge now! Thank you!!! Next up... blog post when i get a chance!!

**Comment by segun-codes** [comment]
> OK, I think we can merge now! Thank you!!! Next up... blog post when i get a chance!!

Oh! okay.. many thanks!

**Comment by segun-codes** [comment]
> OK, sorry that took me a while! I also better understand the question about pointing at github pages vs. the local copy. I opted for GH Pages, so that it's more immediately usable!

Okay.

**Pull request merged**